src/sentry/static/sentry/app/views/organizationEventsV2/modalPagination.jsx

```diff
@@ -87,6 +87,7 @@ const ModalPagination = props => {
 ModalPagination.propTypes = {
   location: PropTypes.object.isRequired,
   event: SentryTypes.Event.isRequired,
+  view: PropTypes.object.isRequired,
```

Comment:
Would be nice to describe this with `shape` instead

Reply:
I might convert this view subtree to typescript later this week so I can do that then.